lib/food_truck.rb

```diff
@@ -0,0 +1,27 @@
+class FoodTruck
+  attr_reader :name, :inventory
+
+  def initialize(name)
+    @name = name
+    @inventory = {}
+  end
+
+  def check_stock(item)
+    inventory[item] ||= 0
+  end
+
+  def stock(item, quantity)
+    inventory[item] ||= 0
+    inventory[item] += quantity
```

Comment:
Yup, this works. You could also have created the hash on line 6 with a default value of 0 and then you could omit the memoization.